example.env

```diff
@@ -33,6 +36,9 @@ export DEFAULT_FROM_EMAIL='admin@datateknologerna.org'
 # Allowed hosts
 export ALLOWED_HOSTS='["0.0.0.0", "127.0.0.1", "localhost"]'
 
+# Allowed origins for csrf
+export ALLOWED_ORIGINS='["https://localhost", "http://localhost"]'
```

Comment:
Should this be set to something else by default?

Reply:
I think it can actually be set to an empty list during development, e.g. []

Reply:
Might be wrong but

Reply:
IIRC it should work with the empty list in development.

I was thinking of having the default values work an example as I don't think they have any effect in development.